### PR TITLE
Feature/mv unused dev

### PR DESCRIPTION
**Pull request by XianwuXue-NOAA**

### CODE REVIEW

File: sorc/build_gefs_ensstat.sh [hunk 1]
```diff
@@ -1,22 +1,21 @@
 #! /usr/bin/env bash
 set -eux
 
-source ./machine-setup.sh > /dev/null 2>&1
+source ./machine-setup.sh #> /dev/null 2>&1
```
Comment (WalterKolczynski-NOAA) [inline sorc/build_gefs_ensstat.sh]: Remove these tail comments if this is the way it is going to stay

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Remove the tail comments where you are no longer redirecting to /dev/null

File: sorc/build_gefs_ensstat.sh (hunk 1 above)
Comment (XianwuXue-NOAA) [reply sorc/build_gefs_ensstat.sh]: OK. I will remove them today after the ecflow meeting. Also I want to modify the versions today.

Comment (XianwuXue-NOAA) [reply sorc/build_gefs_ensstat.sh]: Just removed all tail comments and update the version/gefs.ver.

**Review by WalterKolczynski-NOAA — DISMISSED** [review]

**Comment by XianwuXue-NOAA** [comment]
I just submitted the last commit. Please go ahead to merge it if no objections.

File: versions/gefs.ver [hunk 2]
```diff
@@ -1,8 +1,26 @@
-export gefs_ver=v11.2.0
-export gsm_ver=v12.1.3
-export post_ver=v5.1.2
-export enkf_ver=v2.0.3
-export grib_util_ver=v1.0.1
-export util_shared_ver=v1.0.5
-#export enkf_ver=`cat $NWROOTp1/versions/gfs.ver |grep enkf_ver |awk -F"=" '{print $2}'`
-#export post_ver=`cat $NWROOTp1/versions/gfs.ver |grep post_ver |awk -F"=" '{print $2}'`
+export gefs_ver=v12.0.0.0
```
Comment (WalterKolczynski-NOAA) [inline versions/gefs.ver:1]: This is one too many version levels. GEFS should be v12.0.0

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: versions/gefs.ver (hunk 2 above)
Comment (XianwuXue-NOAA) [reply versions/gefs.ver:1]: Dingchen said "NCO will finalize the 3 section name. A fourth section give us more freedom before NCO finalization"

Comment (WalterKolczynski-NOAA) [reply versions/gefs.ver:1]: Properly, semantic version numbers should be v12.0.0-0 or v12.0.0-beta if that is what he is going for https://semver.org/

Comment (XianwuXue-NOAA) [reply versions/gefs.ver:1]: I will send an email to Dingchen and you to discuss the gefs_ver.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**